draft-birkholz-cose-cometre-ccf-profile.md

```diff
@@ -98,30 +121,35 @@ where:
 - : denotes concatenation of lists
 - D\[k1:k2\] = D'_(k2-k1) denotes the list \{d'\[0\] = d\[k1\], d'\[1\] = d\[k1+1\], ..., d'\[k2-k1-1\] = d\[k2-1\]\} of length (k2 - k1).
 
-## Leaf Components
+## Transaction Components
 
 Each leaf in a CCF ledger carries the following components:
 
 ~~~
-CCF-leaf = [
-  internal-hash: bstr ; a string of HASH_SIZE bytes;
-  internal-data: bstr; a string of at most 1024 bytes; and
-  data_hash: bstr ; the serialization of the element stored at this leaf.
+ccf-leaf = [
+  internal-transaction-hash: bstr ; a string of HASH_SIZE bytes
+  internal-evidence: bstr         ; a string of at most 1024 bytes
+  data-hash: bstr                 ; a string of HASH_SIZE bytes
```

Comment:
Shall `data-hash` be explained below too?

Reply:
done